alternator/streams.cc

```diff
@@ -862,6 +864,12 @@ void executor::add_stream_options(const rjson::value& stream_specification, sche
     }
 
     if (stream_enabled->GetBool()) {
+        auto& db = _proxy.get_db().local();
+
+        if (!db.features().cluster_supports_cdc()) {
+            throw api_error::validation("StreamSpecification: streams (CDC) feature not enabled in cluster.");
```

Comment:
Super Nit: s/in cluster/in the cluster/

Reply:
Actually, no. The use of "the" is "spoken"/"natural" langague. Error reporting should (imho) use the above unspecific form.

Reply:
I'm not sure I follow. So you're saying that "the" is not used in "formal" or "written" language?

Anyways, if you feel like the current version is ok then I don't have any strong feelings about it.

Reply:
No, I am saying error reporting uses mostly unqualified terms. Like "file unreadable". not "the file is unreadable".

Reply:
Ok.